inference-engine/src/vpu/myriad_plugin/myriad_executor.cpp

```diff
@@ -266,15 +266,15 @@ DevicePtr MyriadExecutor::openDevice(std::vector<DevicePtr>& devicePool,
                 return device->isBooted() && device->isNotFull()
                        && device->isSuitableForConfig(config);
             });
-
         // Return mock device. If try infer with it, exception will be thrown
-        if (availableDevices.empty() && config.platform() != NC_ANY_PLATFORM) {
+        if (availableDevices.empty()) {
+            if (config.platform() == NC_ANY_PLATFORM) {
```

Comment:
Do we need https://github.com/openvinotoolkit/openvino/blob/bee21b5c5b3c95d05097df9ccfd95e109b1215e6/inference-engine/src/vpu/myriad_plugin/myriad_config.h#L55 method? 
It has NC_ANY_PLATFORM value by default but is never changed
So it looks for me that this condition is always true

Reply:
I mean, we can possibly remove 
https://github.com/openvinotoolkit/openvino/blob/bee21b5c5b3c95d05097df9ccfd95e109b1215e6/inference-engine/src/vpu/myriad_plugin/myriad_config.h#L90
and  
https://github.com/openvinotoolkit/openvino/blob/bee21b5c5b3c95d05097df9ccfd95e109b1215e6/inference-engine/src/vpu/myriad_plugin/myriad_config.h#L55
and then don't use 
`if (config.platform() == NC_ANY_PLATFORM) {` 
condition

Reply:
done